snapcraft.yaml

```diff
@@ -23,25 +23,25 @@ slots:
 apps:
   gnome-tetravex:
     command: usr/bin/gnome-tetravex
-    extensions: [ gnome-3-38 ]
+    extensions: [ gnome ]
     desktop: usr/share/applications/org.gnome.Tetravex.desktop
 
 parts:
   gnome-tetravex:
     source: https://gitlab.gnome.org/GNOME/gnome-tetravex.git
     source-type: git
     source-tag: '3.38.2'
+    parse-info: [usr/share/metainfo/org.gnome.Tetravex.appdata.xml]
     override-pull: |
-      snapcraftctl pull
-      snapcraftctl set-version $(git describe --tags --abbrev=10)
+      craftctl default
+      patch -p1 < $CRAFT_PROJECT_DIR/patches/patch1.diff
```

Comment:
Consider using *.diff here in case there are other patches. Also please rename the patch to something more intuitive like fix-translations.diff or something.

Reply:
Confirmed: bash wildcards expansion guarantees alphabetical order.

Reply:
But using *.diff doesn't work, bash complains. A for loop must be used.